server/tests/src/test/java/org/infinispan/server/functional/resp/RespScriptTest.java

```diff
@@ -0,0 +1,59 @@
+package org.infinispan.server.functional.resp;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.shouldHaveThrown;
+import static org.infinispan.test.TestingUtil.k;
+import static org.infinispan.test.TestingUtil.v;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import io.vertx.core.Vertx;
+import io.vertx.junit5.VertxTestContext;
+import io.vertx.redis.client.RedisAPI;
+import io.vertx.redis.client.impl.types.ErrorType;
+
+public class RespScriptTest extends AbstractRespTest {
```

Comment:
I believed we had already a test, ScriptingTest, but I'm fine on adding this one